src/Turbo/src/TurboStreamResponse.php

```diff
@@ -104,4 +104,16 @@ public function refresh(?string $requestId = null): static
 
         return $this;
     }
+
+    /**
+     * @param array<string> $attr
```

Comment:
```suggestion
     * @param array<string, scalar> $attr
```

?

Reply:
Scalar includes bool, and I think passing null as value for boolean attributes is better, as per
https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML

So I think `array<string, string|int|float|null>` is the correct type.